static/js/src/advantage/react/components/Subscriptions/Content/Content.tsx

```diff
@@ -1,14 +1,23 @@
 import { Card } from "@canonical/react-components";
 import React from "react";
+import { useMediaQuery } from "react-responsive";
 
 import SubscriptionDetails from "../SubscriptionDetails";
 import SubscriptionList from "../SubscriptionList";
 
-const Content = () => (
-  <Card className="u-no-margin--bottom u-no-padding p-subscriptions__card">
-    <SubscriptionList />
-    <SubscriptionDetails />
-  </Card>
-);
+enum Breakpoint {
```

Comment:
What benefit does media queries in js have over CSS which is where we normally have them?

Reply:
Ah sorry, I should have explained that. On large screens we show the list of subscriptions and the subscription details [side by side](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60ec61de5c8108adc48c822e), but when we switch to mobile view then the functionality changes so that you're either viewing the [subscription list](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60f6fa108d36bb10a7f201a0) or [the details in a modal](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60f6fa1064b2e516c975fbb4) so we need a way to adapt the React code at different screen sizes.

At the moment this just hides the subscription details because selecting a subscription hasn't been implemented yet.

Reply:
I thought we could achieve this by adding a class to the details panel which extends the modal styling at `$breakpoint-small`? This would enforce us to remain consistent with the modal pattern and keep all remaining mark up and react logic the same.

Reply:
We can change the appearance of the details panel to a modal, but we still have to implement some functionality to open and close the modal (to switch between the list and details). In the large screen mode we always have a selected subscription, but in the mobile view we have to change that to be able to not have a selected subscription so that we can display the list.

Reply:
Yes of course there should be a `is-active` class or something to the details panel which would only adopt the modal styling at small screens.